sql-statements/sql-statement-show-indexes.md

```diff
@@ -10,25 +10,14 @@ The statement `SHOW INDEXES [FROM|IN]` lists the indexes on a specified table. T
 
 ## Synopsis
 
-**ShowIndexStmt:**
+```ebnf+diagram
+ShowIndexStmt ::=
+    "SHOW" ( "INDEX" | "INDEXES" | "KEYS" ) ("FROM" | "IN" ) TableName (("FROM" | "IN") SchemaName )? ShowLikeOrWhereOpt?
 
-![ShowIndexStmt](/media/sqlgram/ShowIndexStmt.png)
-
-**ShowIndexKwd:**
-
-![ShowIndexKwd](/media/sqlgram/ShowIndexKwd.png)
-
-**FromOrIn:**
-
-![FromOrIn](/media/sqlgram/FromOrIn.png)
-
-**TableName:**
-
-![TableName](/media/sqlgram/TableName.png)
-
-**ShowLikeOrWhereOpt:**
-
-![ShowLikeOrWhereOpt](/media/sqlgram/ShowLikeOrWhereOpt.png)
+ShowLikeOrWhereOpt ::=
```

Comment:
for consistency with other `SHOW` statements these should be named `ShowLikeOrWhere` too.

Reply:
```suggestion
ShowLikeOrWhere ::=
```